pkg/status/singletonstatus.go

```diff
@@ -63,28 +63,55 @@ func (c *Controller) updateWorkStatusToObject(ctx context.Context, workStatusON
 
 func (c *Controller) syncWorkloadObject(ctx context.Context, wObjID util.ObjectIdentifier) error {
 	logger := klog.FromContext(ctx)
-	requested, nWECs := c.bindingPolicyResolver.GetSingletonReportedStateRequestForObject(wObjID)
+	isSingletonRequested, nWECs := c.bindingPolicyResolver.GetSingletonReportedStateRequestForObject(wObjID)
+	logger.V(4).Info("Workload object (singleton status requested)", "object", wObjID, "isSingletonRequested", isSingletonRequested, "nWECs", nWECs)
+
+	// TODO: GetMultiWECReportedStateRequestForObject is yet to be implemented.
+	// just declaring these variables, we will get its actual value once above function is implemented.
+	var isMultiWECRequested bool
+	logger.V(4).Info("Workload object (multiWEC status requested)", "object", wObjID, "isMultiWECRequested", isMultiWECRequested, "nWECsMulti", nWECs)
```

Comment:
Pro tip: gather all the related information in one log message rather than spread it out among many. You will thank yourself when you read a log from a busy controller working on multiple things concurrently.